eldritch/events.py

```diff
@@ -612,6 +612,87 @@ def finish_str(self):
     return f"{self.character.name} keeps " + ", ".join(self.kept)
 
 
+class Sell(Event):
+
+  def __init__(self, character, target_decks, sell_count=1, discount_type="fixed", discount=0,
```

Comment:
Same idea - let's make this an ItemChoice followed by a Sell. The ItemChoice has a different user interface from MultipleChoice, though I suppose we would have to extend it to include choosing nothing.